### PR TITLE
Add note when replacing keystores during migration 

### PR DESCRIPTION
**Pull request by vithu30**

### CODE REVIEW

File: en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md [hunk 1]
```diff
@@ -1698,6 +1698,25 @@ Follow the instructions below to move all the existing API Manager configuration
 
 5.  Copy the keystores (i.e., `client-truststore.jks`, `wso2cabon.jks` and any other custom JKS) used in the previous version and replace the existing keystores in the `<API-M_3.0.0_HOME>/repository/resources/security` directory.
 
+    !!! Attention
+        In API Manager 3.0.0, it is required to use ceritificate with RSA key size greater than 2048. If you have used a certificate that has weak RSA key (key size less than 2048) in previous version, you need to add following configuration to `<API-M_3.0.0_HOME>/repository/conf/deployment.toml` to configure internal and primary keystores. You should point the internal keystore to the keystore copied from API Manager 2.0.0 and primary keystore can be pointed to a keystore with a ceritificate, which has strong RSA key.
```
Comment (npamudika) [inline en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: API Manager version should be changed as 2.1.0

Comment (vithu30) [reply en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Fixed

Comment (npamudika) [inline en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Small typo in certificate. And shall we change like this.
```suggestion
        In API Manager 3.0.0, it is required to use a certificate with the RSA key size greater than 2048. If you have used a certificate that has a weak RSA key (key size less than 2048) in the previous version, you need to add the following configuration to `<API-M_3.0.0_HOME>/repository/conf/deployment.toml` file to configure internal and primary keystores. You should point the internal keystore to the keystore copied from API Manager 2.0.0 and primary keystore can be pointed to a keystore with a certificate, which has a strong RSA key. 
```

Comment (vithu30) [reply en/docs/InstallAndSetup/UpgradingWSO2APIManager/upgrading-from-210-to-300.md]: Changed as suggested

**Pull request merged**